### ISSUE TITLE
Ship Particle Weapons Double Dip Damage Perks

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#758]
Ship particle weapons are benefiting from the Energy Weapon Systems and Particle Beam Weapon Systems perks. They should only benefit from the latter. Solution; Remove SpaceshipEnergyWeapon [KYWD:0002226A]

---

## Details 
0022ec51,
0022ec52,
0022ec57,
0022ec58,
0022ec5e,
0022ec5f,
0000424c,
0016ad51,
0022ec41,
0022ec42,
0022ec46,
0022ec47,
0022ec4b,
0022ec4c

## Game Version
1.8.86.0

## Language
English (EN)

## New Game
Yes

## Reproduction Steps
Load up xedit and check keywords for said weapons.

**Comment by Constellation-VASCO** [comment]
This does appear to be intended as this only happens with "Electron/Proton" Beam weapons; keep in mind particle turrets also get a double perk buff from eg Particle Weapons and Automated Weapons.

**Comment by Constellation-VASCO** [comment]
Unless I'm mistaken, the most recent patch worked in a change to buff all the other electron/proton weapons in this way. But yeah, removing these energy tags is a different way to achieve consistency. Unless we hear from a dev, I couldn't actually pick a side for which is correct.

**Comment by JaeDLmods** [comment]
The specific EM weapons that benefit from both perks are "disruptor" weapons that are only made by a Ballistics-heavy manufacturer.  Some o fthese were properly set to beneft from laser and particle by default, and Ship Fixes made all of them consistent.

Note that this is not ALL EM weapons.

From the creator of Ship FIxes:
There are 187 references in the Starfield.esm file to the EMP-80 Suppressor.   At a quick glance, almost all of them appear to be ships of some sort.  This is one of several reasons why I believe it was the game designer's intention that the EMP Suppressor weapons are supposed to inflict hull and shield damage in addition to EM damage, because the primary use of EM weapons is to board ships, and NPC ships cannot engage in boarding which negates most of an EM weapon's usefulness.   I believe that the game designers gave EM weapons to NPC ships so that we the players can experience something of what it is like to be targeted by an EM weapon, but then gave the weapons hull/shield damage as well, because otherwise the NPC ships would be severely under-powered in ship combat.  This also follows what I found in the records themselves where there are AMMO/PROJ/EXPL records specifically for the EMP Suppressor weapons to make them different than the other EM weapons.

Also from him:
I wrote a small novel with my set of fixes explaining the reasoning behind the fixes.  While I agree that an EM weapon inflicting hull/shield damage seems odd, when I was looking through the details of the records for the EMP Suppressor weapons it looked like the game designers had actually intended the EMP Supressor weapons to be different than the other EM weapons.  So in line with the overall objective of the patch - namely to correctly implement what appears to have been the intention of the game designers - I made all the EMP Supressor weapons consistent with what appeared to be the original intent.  I mean, I can write a add-on patch mode that would change all the EMP Suppressor weapons to inflict only EM damage.  Just load the patch-of-the-patch after the main ESM files.
